### PR TITLE
feat: rewrite script in js

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

File: scripts/myaegeestatus.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//   https://github.com/linuxbandit
+
+module.exports = (robot) =>
+  robot.hear(/what is the status of (frontend|core|events|statutory|discounts|mailer)/i, (res) => {
+      
+      service = res.match[1]
+      base_url = "https://my.aegee.eu/api"
+
+      robot.http(base_url + `/${service}/healthcheck`)
+      .header('Accept', 'application/json')
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:25]: can you add 2 more spaces for this line and later? so it'd be a bit more readable?

File: scripts/myaegeestatus.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      .get() ( (err, response, body) => {
+
+        // err & response status checking code here
+        if (err != null || response.headers['content-type'].indexOf('application/json') === -1){
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:29]: space before {

File: scripts/myaegeestatus.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          robot.logger.error(err)
+          return res.send("An error occurred, or the answer was not JSON :(")
+        }
+
+        data = null
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:34]: you can also remove it and use `const data = JSON.parse(body)` later

File: scripts/myaegeestatus.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        try{
+          data = JSON.parse(body)
+          health = ':warning: UNHEALTHY :warning:'
+          if(data.success)
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:38]: it's better to wrap if blocks with {}, even if there is 1 line of code inside, it'd be more readabble

File: scripts/myaegeestatus.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            health = 'HEALTHY :ok:'
+          return res.send(`Service ${service} is ${health}`)
+        }
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:41]: `} catch (error)`

File: scripts/myaegeestatus.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+        .get() ( (err, response, body) => {
+
+        // err & response status checking code here
+        if (err != null || response.headers['content-type'].indexOf('application/json') === -1) {
+          robot.logger.error(err)
+          return res.send("An error occurred, or the answer was not JSON :(")
+        }
+
+        let data = null
+        try{
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:36]: try {

File: scripts/myaegeestatus.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+          data = JSON.parse(body)
+          let health = ':warning: UNHEALTHY :warning:'
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:38]: you can use ternary here, like

```
const health = data.success
    ? 'HEALTHY :ok:'
    : ':warning: UNHEALTHY :warning:';
```

File: scripts/myaegeestatus.js [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+            health = 'HEALTHY :ok:'
+          }
+          return res.send(`Service ${service} is ${health}`)
+        } catch (error) {
+            robot.logger.error(error)
+            return res.send("Ran into an error parsing JSON :(")
+        }
+      })
+  })
+
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:48]: no need for newline here (also inthe line 20

File: scripts/myaegeestatus.js (hunk 7 above)
Comment (linuxbandit) [reply scripts/myaegeestatus.js:37]: lol omg true! i was like 'fuck this shit' with coffeescript that i now forgot how to :D

File: scripts/myaegeestatus.js (hunk 8 above)
Comment (linuxbandit) [reply scripts/myaegeestatus.js:48]: it's fine, later I add another listener

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**